Gems/GradientSignal/Code/Include/GradientSignal/Components/ImageGradientComponent.h

```diff
@@ -186,6 +186,7 @@ namespace GradientSignal
         void GetValuesInternal(SamplingType samplingType, AZStd::span<const AZ::Vector3> positions, AZStd::span<float> outValues) const;
         float GetValueFromImageData(SamplingType samplingType, const AZ::Vector3& uvw, float defaultValue) const;
         float GetPixelValue(AZ::u32 x, AZ::u32 y) const;
+        float InvertHeightAndGetPixelValue(AZ::u32 x, AZ::u32 y) const;
```

Comment:
suggestion: An API comment for this call might be useful to make it clear what it's doing - it might be clearer to just say `GetInvertedPixelValue` perhaps? Or maybe there's more to it than that?

Reply:
The method is private to the component, but I'll add an API comment. I can't use "GetInvertedPixelValue" because that makes it sound like the pixel value is getting inverted, not the Y component for lookup. 

The backstory here is that there's a tiny chunk of code in GetPixelValue() that I wanted to avoid duplicating multiple times in this file. But half the places that use it need to invert the Y value for lookup and half don't because they've already got an inverted Y. So I split this into two methods, one that inverts the Y first and calls GetPixelValue(), and then just GetPixelValue() itself.

I also considered "InvertYAndGetPixelValue" but that just hurts my eyes a little bit, I find it harder to read that "Y" quickly next to the capital A. I'd be willing to switch to this though if you think it's more clear.

Reply:
Changed name and added API comments.

Reply:
Thanks for the description and adding the comment @mbalfour-amzn, much appreciated!